### ISSUE TITLE
WiFi may cause kernel crash.

### ISSUE THREAD

**Issue by ssdsl0126** [opa334/Dopamine#156]
When Wi-Fi is in a state between connection and disconnection (weak signal), it is very easy to cause a kernel crash.
[panic-full-2023-05-11-141812.000.log](https://github.com/opa334/Dopamine/files/11449888/panic-full-2023-05-11-141812.000.log)

**Comment by opa334** [comment]
Can you reliably trigger this?

**Comment by Gay-Blakey** [comment]
I've been able to trigger this by accident. By using a vpn(mullvad), of which, throttles my connectivity due to it's strict focus on privacy, does indeed cause the kernel to crash. Although this is just a hunch. Also correlates with the wifid doing it's simulated crashes as well. If I had an app or a cli script or commands  to start the wifid simulated crash while on a vpn or be far enough from my router, I could see if I indeed panic every time or not. Crash logs could be provided, if needed. Currently testing to see if while my vpn is off and have 100% connectivity to my router, do I crash? And while away from my house, I'll have my wifi off to see if I crash or not. Will report back and provide crash logs if something happens.
